pkg/codec.go

```diff
@@ -351,6 +375,11 @@ func processAttachments(decoder iface.Decoder, message remote.Message) error {
 	if err != nil {
 		return err
 	}
+	//
+	//if attachmentsRaw == nil || attachmentsRaw == "" {
+	//	attachmentsRaw = map[interface{}]interface{}{"interface": service.}
+	//}
```

Comment:
Will we use the code commented out later? Seems unlikely, the values have different types.